mkt/access/migrations/0003_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
It just a merge migration because 0002_group_restricted also depended on 0001_... (it was created before 0002_auto... existed).  Django hassled me about creating it - should I attempt to manually merge it?